content/spin/v2/python-components.md

```diff
@@ -380,3 +497,20 @@ For more information about using SQLite from Python, see [SQLite storage](sqlite
 ## AI Inferencing From Python Components
 
 For more information about using Serverless AI from Python, see the [Serverless AI](serverless-ai-api-guide) API guide.
+
+## Troubleshooting
+
+If you bump into issues when installing the requirements.txt. For example:
+
+<!-- @nocpy -->
+
+```console
+error: externally-managed-environment
+× This environment is externally managed
+```
+
+Please note, this error occurs because installing a **non-brew-packaged** Python package requires you to either:
```

Comment:
```suggestion
Please note, this error is specific to Homebrew-installed Python installations and occurs because installing a **non-brew-packaged** Python package requires you to either:
```